core/src/main/java/bisq/core/dao/governance/votereveal/VoteRevealService.java

```diff
@@ -83,15 +79,10 @@
     private final BtcWalletService btcWalletService;
     private final P2PService p2PService;
     private final WalletsManager walletsManager;
-
     @Getter
     private final ObservableList<VoteRevealException> voteRevealExceptions = FXCollections.observableArrayList();
     private final List<VoteRevealTxPublishedListener> voteRevealTxPublishedListeners = new ArrayList<>();
-
-    ///////////////////////////////////////////////////////////////////////////////////////////
-    // Constructor
-    ///////////////////////////////////////////////////////////////////////////////////////////
-
+    private final List<VoteRevealTxFailedListener> voteRevealTxFailedListeners = new ArrayList<>();
```

Comment:
NACK - You have to add  an empty line before the @Inject to apply our style guide.  I would have pushed this fix myself, but it seems to be that you are not allowing maintainers to push to your PR. Also please create a PR next time based on a separate branch and not directly from your master branch. Thanks!

Reply:
Apologies. Even **`@Getter`** definition was affected. Fixed.